guide/payments/send.md

```diff
@@ -39,169 +103,255 @@ layout = "full-width"
 
-Human error with fee selection can lead to costly mistakes, and fee estimations are imperfect. Carefully consider if and how you expose transaction fee selection to senders.
 </div>
 
-**Variations**
-- Automatically set the fee rate
-- Allow sender to choose from presets
-- Allow sender to enter a custom fee rate (advanced)
+## Transaction processing
+
+Processing times may also differ a lot between on-chain and Lightning network payments. On-chain, pending transactions are bundled into a [new block]({{ '/guide/getting-started/technology-primer/#what-is-a-blockchain' | relative_url }}) roughly every 10 minutes. On the Lightning network, [payment routing]({{ '/guide/getting-started/technology-primer/#how-are-payments-routed' | relative_url }}) happens instantly and is largely dependent on the number of nodes involved and their responsiveness.
+
+When transactions take longer than expected, users need to be clearly informed about the status. In scenarios like in-store payments, speedy confirmation is of the essence, as the user wants to move on, and the merchant may have other customers waiting. In-app status updates can be coupled with notifications, or even emails, to ensure that both parties are confident that everything is in order.
+
+<div class="image-slide-gallery">
+
+{% for item in page.imagesProcessing %}
+
+{% capture imageURL %}{{ page.image_base }}{{ item.file }}.png{% endcapture %}
+{% capture imageURLRetina %}{{ page.image_base }}{{ item.file }}@2x.png{% endcapture %}
+
+{% include picture.html
+   image = imageURL
+   retina = imageURLRetina
+   alt-text = item.alt
+   caption = item.caption
+   width = 250
+   height = 541
+   layout = "shadow"
+%}
+{% endfor %}
+
+</div>
 
-**Do's**
+In case a user needs to briefly wait, the application should not block the interface, but offer other useful actions to perform, such as labelling and tagging the payment, or adding the recipient to the contact list.
 
-- If the sender does not have advanced knowledge about bitcoin fees but wants control over the fee rate, provide presets to select how urgent the transaction is (e.g. low, medium, high priority)
-- When using presets, clearly communicate estimated confirmation time and costs related to each of the fee rate options
-- When allowing users to enter their own fee rate, warn the sender if they enter an amount far beyond the recommended fee rate for the next block
+On-chain, it may also be possible to cancel (via [replace-by-fee](https://bitcoinops.org/en/topics/replace-by-fee/) or double-spend with a higher fee) or speed-up (via [child-pays-for-parent](https://bitcoinops.org/en/topics/cpfp/)) a transaction after it has been broadcast, but before it has been included in a block.
```

Comment:
```suggestion
On-chain, you may want to offer your users the option to cancel (via [replace-by-fee (RBF)](https://bitcoinops.org/en/topics/replace-by-fee/)) or speed-up (via [child-pays-for-parent (CPFP)](https://bitcoinops.org/en/topics/cpfp/)) a transaction after it has been broadcast, but before it has been included in a block.
```

With points like this I think we should speak to the reader like above. Does double spending cancel or does the higher fee just get confirmed?

Reply:
`or double-spend with a higher fee` is an explanation of what `replace by fee` does. You just make another transaction with the same coins but a higher fee, so it gets confirmed faster than the first one. The first one remains in the mempool, but will be rejected by the network because the coins have then been spent by the second transaction.

I may need to revise my writing with this active language that directly addresses the reader. Have mostly tried to stay away from it.

Reply:
Rewrote this part slightly. We should add RBF and CPFP to the glossary with simple descriptions, and link to there.

Reply:
Created an issue for adding the two terms to the Glossary.